tools/mo/openvino/tools/mo/serialize.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from openvino.tools.mo.back.ie_ir_ver_2.emitter import append_ir_info
+from openvino.tools.mo.utils.cli_parser import get_meta_info
+from openvino.tools.mo.utils.error import Error
+from openvino.tools.mo.utils.telemetry_utils import get_tid
+from openvino.tools.mo.utils.utils import refer_to_faq_msg
+from openvino.tools.mo.utils.version import get_simplified_mo_version
+
+try:
+    import openvino_telemetry as tm
+except ImportError:
+    import openvino.tools.mo.utils.telemetry_stub as tm
+
+
+def serialize(ngraph_function, xml_path, argv=None):
```

Comment:
@popovaan, I heard there has been already implemented ov.serialize function (@ilya-lavrenov for confirmation). Could we just call it? Are we providing extended functionality over ov.serialize? If not, I propose deleting mo.serialize.

Reply:
> ov.serialize function

CC @AlexeyLebedev1

Reply:
I call serialize() from openvino.runtime here. 
The only extended functionality is appending MO params info to IR.

Reply:
mo.serialize deleted.